wpe/src/main/cpp/browser/browser.cpp

```diff
@@ -14,85 +15,39 @@
 #include <android/surface_control.h>
 
 #define PAGE_METHOD_PROXY(BROWSER_METHOD, PAGE_METHOD, PRIORITY) \
-    void Browser::BROWSER_METHOD(int pageId) \
-    { \
-        if (m_pages.find(pageId) == m_pages.end()) \
-            return; \
-        g_main_context_invoke_full(*m_uiProcessThreadContext, PRIORITY, +[](gpointer data) -> gboolean { \
-            auto* page = reinterpret_cast<Page*>(data); \
-            if (page != nullptr) page->PAGE_METHOD(); \
-            return G_SOURCE_REMOVE; \
-        }, m_pages[pageId].get(), nullptr); \
+    void Browser::BROWSER_METHOD(int pageId)                     \
```

Comment:
In the future we may think about creating a variadic template, so we may use it with all methods, including the ones with parameters.

Reply:
Agree, let's do it later